### ISSUE TITLE
Farming Tracker doesn't track deposited materials

### ISSUE THREAD

**Issue by gooseears** [gw2efficiency/issues#1289]
Farming Tracker working for character bags and bank, but as soon as you deposit to material storage, it stops tracking it, like it was never farmed.

**Comment by queicherius** [comment]
Hey there, it should definitely still track them in the storage. Keep in mind that it can take the API up to 10 minutes to catch up to the change - did you allow for this time?

**Comment by gooseears** [comment]
I definitely waited at least 10 minutes.  Here is what I am seeing - https://i.imgur.com/SEu9Ffw.png

Odd thing is it isn't updating either my material storage or my bank, but my characters' bags are updating no problem.

**Comment by gooseears** [comment]
Okay, I just removed all of my API keys from my arenanet account and created a new one and put that into gw2efficiency.  Everything seems to be working now.  Something must have happened to my API keys in the 4 month break I took from this game.  Closing the ticket.

Thanks.

**Comment by queicherius** [comment]
Yeah from your image description (thanks a lot for going through the trouble to be that detailed by the way!) it looks like it was an API issue. Glad to hear it's working now!

**Comment by gooseears** [comment]
Hm, it happened again.  Bank and material storage isn't updating, but my character's bags are just fine.  I did leave my farming tracker on for like 8 hours yesterday, maybe that was too many calls yesterday.  Do they rate limit the material storage and bank endpoint after a certain number of calls?   Why don't they rate limit the character bag endpoint?

**Comment by queicherius** [comment]
> Bank and material storage isn't updating, but my character's bags are just fine. 

Did you confirm that on other parts of the site (= does the change not show up on the material storage page either?)

> Do they rate limit the material storage and bank endpoint after a certain number of calls?

There is a rate limit in place, but it's for the entire API.

**Comment by gooseears** [comment]
Yeah I checked the bank and material storage pages, and they were not updating.  But my characters' bags definitely were, because the farming tracker was still going.

I'm thinking I hit some kind of limit.  I stopped leaving it on, hitting the endpoints every 3 minutes must have done something.  The only way I could fix it was to delete the entire API key and make a new one.  Rate limit over the entire API makes sense, but I'm convinced something else is happening on anet's side with my API key.

I'm still using the farming tracker, but I'm just leaving it paused and only resuming it once in a while so I don't hit that limit again.
